src/custom-element.js

```diff
@@ -707,6 +704,7 @@ function createBaseCustomElementClass(win) {
         this.implementation_.layoutWidth_ = this.layoutWidth_;
       }
       // TODO(malteubl): Forward for stubbed elements.
+      // TODO(jridgewell): We should pass the layoutBox down.
```

Comment:
`this.implementation_.onLayoutMeasure();`. We should pass it to `#onLayoutMeasure`, because a lot of them immediately call `#getLayoutBox`.